STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.examiner.cdi.ExaminerInjectTest.testFindEventNamedClass
MODIFIED FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/ExaminerInjectTest.java
CLEANER: de.strullerbaumann.visualee.examiner.cdi.TestGroup100Case0.test259
CLEANER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 16
NEW CLEANER SIZE: 2
ELAPSED TIME: 106.85500001907349
==========================
@@ -174,0 +174,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
